infra/src/test/kotlin/fi/fta/geoviite/infra/split/SplitTestDataService.kt

```diff
@@ -41,12 +40,29 @@ class SplitTestDataService @Autowired constructor(
     private val splitService: SplitService,
 ) : DBTestBase() {
 
+    fun clearSplits() {
```

Comment:
Ajettaessa IT-testi useammin kuin kerran aiheutui constraint-ongelmia ellei myös splittitauluja tyhjennellyt välissä.